glotaran/project/project.py

```diff
@@ -0,0 +1,508 @@
+"""The glotaran project module."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from dataclasses import field
+from pathlib import Path
+from textwrap import dedent
+from typing import Any
+from typing import Literal
+
+import xarray as xr
+
+from glotaran.builtin.io.yml.utils import load_dict
+from glotaran.model import Model
+from glotaran.parameter import ParameterGroup
+from glotaran.project.project_data_registry import ProjectDataRegistry
+from glotaran.project.project_model_registry import ProjectModelRegistry
+from glotaran.project.project_parameter_registry import ProjectParameterRegistry
+from glotaran.project.project_result_registry import ProjectResultRegistry
+from glotaran.project.result import Result
+from glotaran.project.scheme import Scheme
+from glotaran.utils.io import make_path_absolute_if_relative
+from glotaran.utils.ipython import MarkdownStr
+
+TEMPLATE = """version: {gta_version}
+"""
```

Comment:
One more nitpick 😅
```diff
- TEMPLATE = """version: {gta_version}
- """
+ TEMPLATE = "version: {gta_version}"
```
Also since it is only used in one place it wouldn't need to be a global variable.

Reply:
Done. As discussed, in future there will be more added so we leave it as global.